### PR TITLE
Merge

### PR DESCRIPTION
**Pull request by lloydsargent42**

### CODE REVIEW

File: boards/arm/nucleo_f429zi/nucleo_f429zi.dts [hunk 1]
```diff
@@ -61,6 +61,7 @@ arduino_serial: &usart6 {};
 
 &spi1 {
 	status = "ok";
+	cs-gpios = <&gpioa 4 0>;
```
Comment (erwango) [inline boards/arm/nucleo_f429zi/nucleo_f429zi.dts]: cs-gpio depends on the way you add hardware to the board and is specific to your need, we cannot take this modification.
Please remove

File: drivers/pinmux/stm32/pinmux_stm32f4.h [hunk 2]
```diff
@@ -108,11 +108,15 @@
...
 	(STM32_PINMUX_ALT_FUNC_9 | STM32_OPENDRAIN_PULLUP)
 
+#define STM32F4_PINMUX_FUNC_PB5_SPI1_MOSI    \
+	(STM32_PINMUX_ALT_FUNC_5 | STM32_PUSHPULL_NOPULL)
 #define STM32F4_PINMUX_FUNC_PB5_I2S3_SD  \
 	(STM32_PINMUX_ALT_FUNC_6 | STM32_PUSHPULL_NOPULL)
 #define STM32F4_PINMUX_FUNC_PB5_UART5_RX    \
 	(STM32_PINMUX_ALT_FUNC_11 | STM32_PUSHPULL_NOPULL)
 
+
```
Comment (erwango) [inline drivers/pinmux/stm32/pinmux_stm32f4.h]: Remove these additional blank lines

**Review by erwango — CHANGES_REQUESTED** [review]
@lloydsargent42 thanks for this pull-request change.
Though, there are several modifications that should be made to get your proposal approved:
-Provide a meaningful Title to the PR
-Add a signed-off-by in you commit
-Clean the 2 last commits, which do not belong here

**Comment by lloydsargent42** [comment]
I've been asked to undo this pull request but I'm a little puzzled if I do it on my end or you do it on your end.

**Comment by erwango** [comment]
@lloydsargent42 , I advise you to look at https://docs.zephyrproject.org/latest/contribute/contribute_guidelines.html#contribution-workflow and follow process step by step.
This is not a big change, so I think it will be easier to start again from scratch.

**Comment by erwango** [comment]
@lloydsargent42  @thescruba , do you plan to do some clean up on this change?

**Comment by lloydsargent42** [comment]
No, it was accidental.

Feel free to remove them.

**Comment by erwango** [comment]
Ok, I'm closing then. Txs for feedback

**Comment by lloydsargent42** [comment]
My apologies for the trouble. I’m still a novice at git.

**Pull request closed without merging**